Cargo.toml

```diff
@@ -29,7 +29,7 @@ default-target = "x86_64-unknown-linux-gnu"
 targets = []
 
 [workspace.lints]
-# LINEBENDER LINT SET - Cargo.toml - v2
+# LINEBENDER LINT SET - Cargo.toml - v4
```

Comment:
Isn't the latest version v5? I think I saw a vello PR for this?

Reply:
Either way I'm satisfied updating to v4, can revisit later.